packages/image_picker/image_picker/pubspec.yaml

```diff
@@ -2,7 +2,7 @@ name: image_picker
 description: Flutter plugin for selecting images from the Android and iOS image
   library, and taking new pictures with the camera.
 homepage: https://github.com/flutter/plugins/tree/master/packages/image_picker/image_picker
-version: 0.7.5+1
+version: 0.7.6
```

Comment:
Since there's no public API change, it should be 0.7.5+2

Reply:
done.